homeassistant/components/fritz/strings.json

```diff
@@ -0,0 +1,43 @@
+{
+  "config": {
+    "flow_title": "FRITZ!Box Tools: {name}",
+    "step": {
+      "confirm": {
+        "title": "Setup FRITZ!Box Tools",
+        "description": "Discovered FRITZ!Box: {name}\n\nSetup FRITZ!Box Tools to control your {name}",
+        "data": {
+          "username": "Username",
+          "password": "Password"
+        }
+      },
+      "start_config": {
+        "title": "Setup FRITZ!Box Tools - mandatory",
+        "description": "Setup FRITZ!Box Tools to control your FRITZ!Box.\nMinimum needed: username, password.",
+        "data": {
+          "host": "Host",
+          "port": "Port",
+          "username": "Username",
+          "password": "Password"
+        }
+      },
+      "reauth_confirm": {
+        "title": "Updating FRITZ!Box Tools - credentials",
+        "description": "Update FRITZ!Box Tools credentials for: {host}.\n\nFRITZ!Box Tools is unable to log in to your FRITZ!Box.",
+        "data": {
+          "username": "Username",
+          "password": "Password"
+        }
+      }
+    },
+    "abort": {
+      "already_in_progress": "FRITZ!Box configuration is already in progress.",
+      "already_configured": "This AVM FRITZ!Box is already configured.",
+      "reauth_successful": "FRITZ!Box configuration successfully updated."
```

Comment:
We should use the standard strings here:
`[%key:common::config_flow::abort::already_in_progress%]` etc

Reply:
Done